pkg/hs/urbit-king/lib/Urbit/Vere/Pier.hs

```diff
@@ -360,6 +361,29 @@ pier (serf, log) vSlog startedSig = do
   tExec <- acquireWorker "Effects" (router slog (readTQueue executeQ) drivz)
   tDisk <- acquireWorkerBound "Persist" (runPersist log persistQ execute)
 
+  -- Now that the Serf is configurated, the IO drivers are hooked up, their
+  -- starting events have been dispatched, and the terminal is live, we can now
+  -- handle injecting events requested from the command line.
+  unless (null injected) $ rio $ do
+    for_ (zip [1..] injected) $ \(num, ev) -> do
+      logTrace $ display @Text ("Injecting event " ++ (tshow num) ++ " of " ++
```

Comment:
you're aware that logTrace is actually the HIGHEST priority, right?

Reply:
Yes. This is to match the printing of major events like "Resume Ship" above. Do you disagree that this warrants this amount of loudness?

Reply:
For each event? What if there are a lot?

Reply:
Then it will give you sufficient debugging information about which event in the sequence errored.